packages/multimedia/kodi/build

```diff
@@ -112,7 +113,7 @@ cmake \
   -DKODI_DEPENDSBUILD=0 \
   -DWITH_CPU=$TARGET_CPU \
   -DWITH_ARCH=$TARGET_ARCH \
-  -DCORE_SYSTEM_NAME=$TARGET_NAME \
+  -DCORE_PLATFORM_NAME=$TARGET_NAME \
```

Comment:
I have a doubt for this , cause if we use gbm, we force the use , ie : 
if pkg_uses $1 gbm ; then
KODI_FEATURES="$KODI_FEATURES -DCORE_PLATFORM_NAME=gbm"